setup.py

```diff
@@ -74,7 +74,7 @@
         "eth-hash[pycryptodome]>=0.5.1",
         "eth-typing>=3.0.0",
         "eth-utils>=2.1.0",
-        "hexbytes>=0.1.0",
+        "hexbytes>=0.1.0,<=0.3.1",
```

Comment:
See comment [here](https://github.com/ethereum/eth-account/pull/240). I think we should use `<0.4.0` as the pin if that is the planned version with the breaking changes.